x-pack/plugins/cloud_security_posture/public/components/take_action.tsx

```diff
@@ -169,6 +170,7 @@ export const TakeAction = ({
   createRuleFn,
   enableBenchmarkRuleFn,
   disableBenchmarkRuleFn,
+  isRuleMuted = true,
```

Comment:
if this boolean fallbacks to true it breaks the Findings Flyout

![image](https://github.com/elastic/kibana/assets/19270322/0f486113-3a76-45f0-8088-df29ae09091c)


Also, `isRuleMuted` is very generic and also doesn't directly imply it will disable the detection rules button, I suggest changing it to something like: `isCreateDetectionRuleDisabled` and fallback it to `false`.

Reply:
woops